packages/frontend/scss/login/_login.scss

```diff
@@ -195,6 +195,7 @@ div.delta-form-group {
     font-family: inherit;
     box-shadow: none !important;
     resize: vertical;
+    field-sizing: content;
```

Comment:
Note that this also affects the "edit profile" dialog:

<img width="477" height="416" alt="Image" src="https://github.com/user-attachments/assets/721baa6d-d915-451e-88b3-bad02dc9ccdd" />

Looks fine IMO.